src/Pages/Profile/admin/EditorForm.js

```diff
@@ -67,7 +67,31 @@ export default function EditForm (props) {
                 style={{ marginTop: '5px' }}
               >
                 Reset Pages!
-              </Button>
+              </Button> */}
+              <FormGroup check inline style={{ marginTop: '5px' }}>
```

Comment:
can you make the style be inside of a `.css` file? you can then assign the FormGroup a classname